README.md

```diff
@@ -1,2 +1,227 @@
+<br></br>
+    
+```
+qne application create [OPTIONS] APPLICATION_NAME ROLES...
+
+Arguments:
+  APPLICATION_NAME  Name of the application  [required]
+  ROLES...          Names of the roles to be created  [required]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne application create application_name role_name1 role_name2
+```
+</details>
+
+
+<!--- QNE APPLICATION DELETE --->
+<details closed>
+<summary><b>qne application delete</b></summary>
+Used to delete an application. Will delete the entire application directory structure.
+<br></br>
+    
+```
+qne application delete [OPTIONS] APPLICATION_NAME
+
+Arguments:
+  APPLICATION_NAME  Name of the application  [required]
+
+Options:
+  --help  Show this message and exit.
+
+Example:
+  qne application delete application_name
+```
+</details>
+
+
+
+<!--- QNE APPLICATION LIST --->
+<details closed>
+<summary><b>qne application list</b></summary>
+Show a list of all existing applications and the path to where they are stored.
+<br></br>
+    
+```
+qne application list [OPTIONS]
+
+Options:
+  --local  List local applications  [default: False].
+  --help   Show this message and exit.
+
+Example:
+  qne application list
+```
+</details>
+
+
+
+<!--- QNE APPLICATION VALIDATE --->
+<details closed>
+<summary><b>qne application validate</b></summary>
+This command can be used to validate the files that are in the application directory. It checks for a correct file structure, if all files and directories needed exist and if the json files are in correct format.
+<br></br>
+    
+```
+qne application validate [OPTIONS]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne application validate
+```
+</details>
+
+
+
+<!--- QNE EXPERIMENT CREATE --->
+<details closed>
+<summary><b>qne experiment create</b></summary>
+Create a new experiment, based on an application name and a chosen network.
+<br></br>
+    
+```
+qne experiment create [OPTIONS] EXPERIMENT_NAME APPLICATION_NAME NETWORK_NAME
+
+Arguments:
+  EXPERIMENT_NAME   Name of the experiment.  [required]
+  APPLICATION_NAME  Name of the application.  [required]
+  NETWORK_NAME      Name of the network to be used. [required]
+
+Options:
+  --local  Run the application locally  [default: True]
+  --help   Show this message and exit.
+  
+Example:
+  qne experiment create experiment_name application_name europe
+```
+</details>
+
+
+
+<!--- QNE EXPERIMENT VALIDATE --->
+<details closed>
+<summary><b>qne experiment validate</b></summary>
+Validates whether the experiment file structure is  complete and if the json content is valid.
+<br></br>
+
+```
+qne experiment validate [OPTIONS]
+
+Options:
+  --help  Show this message and exit.
+  
+Example:
+  qne experiment validate
+```
+</details>
+
+
+<!--- QNE EXPERIMENT DELETE --->
+<details closed>
+<summary><b>qne experiment delete</b></summary>
+Delete the entire experiment.
+<br></br>
+    
+```
+qne experiment delete [OPTIONS] EXPERIMENT_NAME
```

Comment:
EXPERIMENT_NAME also optional. Why not copy the latest version from the current qne-adk?